GoodDollar/GoodDAPP
# (Feature) integrate app with zoom

# Description

[x] Add Zoom API
[x]  add hook for initialize ZoomSDK
[x]  add hook for perform Zoom verificartion session
[x]  implement FaceVerification.web component using hooks written and Zoom UI 'out-of-box'
[x]  configure error handling & redirection on success

About #1537 


# How Has This Been Tested?

1. Open app on web
2. Create new account
3. Click to Claim button
4. Should be redirected on FaceVerification page
5. Zoom UI loads, camera view should be on senter
6. Press "i'm ready" button
7. Follow Zoom instructions
8. Await face snapshots uploaded
9. You should got success message and redirected to Dashboard
10. Then you should be able to claim


# Checklist:
